apps/site/pages/docs/migration-guides/Migrating to 2.2.65 - graphql-utils depreciation.mdx

```diff
@@ -0,0 +1,141 @@
+# Migrating your store to v2.2.65
```

Comment:
```suggestion
# Improvements to API extensions 
```

Reply:
@icazevedo, @hellofanny and I were talking about the word "Migrating" in this context, and maybe it can cause some misunderstanding. Maybe people will think we are talking about a new FastStore version and not exactly the faststore/core package. What do you think about this new title suggestion?